### DISCUSSION TITLE
Stablizerのref_foot_orogin_*

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#28] [Q&A] [answered]
Stablizerにref_foot_orogin_*をセットするのはここですが、
https://github.com/YutaKojio/hrpsys-base/blob/5aab7f9809b1ef76dd62bf323e522834d84097be/rtc/AutoBalancer/AutoBalancer.cpp#L1095
それよりも前の場所でref_foot_origin_*を用いてcogを求める処理が行われるので、
https://github.com/YutaKojio/hrpsys-base/blob/5aab7f9809b1ef76dd62bf323e522834d84097be/rtc/AutoBalancer/AutoBalancer.cpp#L1241
初回などにバグになる恐れがありそう?

**Comment by YutaKojio** [comment, marked as answer]
初回から歩くコマンドは呼ばれてないだろうという前提の部分がちょこちょこあるかも
